media/css/pebbles/components/_protocol-footer.scss

```diff
@@ -0,0 +1,58 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+$font-path: '/media/fonts';
+$image-path: '/media/protocol/img';
+
+// Protocol components for footer.
+@import '../../../protocol/css/components/footer';
+@import '../../../protocol/css/components/language-switcher';
+
+/**
+ * Pebbles specific overrides for Protocol Footer component.
+ * 1. Pebbles pages use Zilla for body copy, so override in the footer.
+ * 2. Because of 1, pages also set a base font-size of 112.5% on <html>, so compensate proportionally.
+ */
+
+@import '../includes/lib';
+
+.mzp-c-footer {
+    @include open-sans;
+
+    .mzp-l-content {
+        @include border-box;
+        margin: 0 auto;
+        padding: 20px;
+
+        @media #{$mq-tablet} {
+            padding: 20px $tablet-content-padding;
+        }
+
+        @media #{$mq-desktop} {
+            padding: 20px 85px;
+        }
+
+        @media #{$mq-desktop-wide} {
+            max-width: $width-max-content;
+        }
+    }
+
+    .mzp-c-footer-heading {
+        @include font-size(16px);
+    }
+
+    .mzp-c-footer-primary {
+        @include font-size(16px);
+    }
+
+    .mzp-c-footer-secondary {
+        @include font-size(12px);
+    }
+}
+
+
+// bug fix, remove after backported to Protocol
```

Comment:
Nice catch, shall we file an protocol issue to make sure we don't forget this?

Reply:
"we" 😝

https://github.com/mozilla/protocol/issues/317